ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -109,6 +109,20 @@ tests:
         VIP_DHCP_ALLOCATION=no
         PROXY=yes
     workflow: baremetalds-assisted-install
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-single-node
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
```

Comment:
@osherdp will this work in this repo?

Reply:
/hold until we make sure it's actually taking the latest image here

Reply:
Don't think it's different from  `e2e-metal-assisted-ipv6`

Reply:
yes, it should
and it will rehearse anyway so we can check

Reply:
Oh right, didn't realize the rest also had it. So it's probably safe (unless they were all not really running on PRs all this time, but on the latest instead)